java/swagger-petstore/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM openjdk:8-jre-alpine
```

Comment:
is docker part of requirements? if we need, is it tested? if not remove

Reply:
It has been tested! Need to validate if it's necessary.